### PR TITLE
Feature/ninty nine/about-page-design

### PR DESCRIPTION
**Pull request by diautzi**
### Describe the problem being solved:
*changes made to center Kim's photo, and added thin border around all photos

### Impacted areas in the application: 
*Before:
Screen Shot 2019-11-04 at 7 12 16 PM
Screen Shot 2019-11-04 at 7 12 29 PM

*After:
Screen Shot 2019-11-04 at 7 06 10 PM
Screen Shot 2019-11-04 at 7 06 25 PM

List general components of the application that this PR will affect: 
* frontend/src/components/static_pages/about.css
* frontend/src/components/static_pages/about.jsx

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts
- [x] I have run the [prettier](https://prettier.io/) command `make pretty`

### CODE REVIEW

File: frontend/src/components/static_pages/about.css [hunk 1]
```diff
@@ -11,15 +11,30 @@
...
 
 .images img {
-  width: 300px;
+ width: 330px;
+ padding: 20px;
+ border: 1px solid #d3d3d3;
+}
+
+#kim-image {
+ margin-top: 60px;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/about.css]: we need to just have this for non mobile environments. In mobile screens the image moves on top and this margin is causing a bit of an issue 
image

File: frontend/src/components/static_pages/about.css [hunk 2]
```diff
@@ -11,15 +11,30 @@
...
-  margin-bottom: 40px;
-  margin-left: 120px;
+ margin: 0;
+ margin-bottom: 40px;
+ margin-left: 120px;
 }
 
 .images img {
-  width: 300px;
+ width: 330px;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/about.css]: also a bit of an issue around the 1100px breakpoint 
image

**Comment by diautzi** [comment]
Fixed the settings for the tablet view on about page

File: frontend/src/components/static_pages/about.css [hunk 3]
```diff
@@ -46,9 +62,15 @@
     }
   }
 
+  #kim-tablet {
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/about.css]: I dont think we need to create and use this id. There is a slightly simpler way.

We can just do this using `kim-image` 

if we change its `margin-top: 20px` inside of this media query -> it should solve the issue! 

There is also a lot of space in-between Our People and Kim's image, and this solution will fix that as well 👍

**Comment by diautzi** [comment]
👍

**Comment by diautzi** [comment]
Should I do it, or you took care of it already?

**Comment by dkaplan88** [comment]
I didn't do it! Left it for you 😛

**Comment by diautzi** [comment]
😆

**Comment by diautzi** [comment]
On it👌🏻

**Comment by diautzi** [comment]
It fixed it!! Thanks

**Pull request merged**